docs/site/Database-migrations.md

```diff
@@ -127,6 +127,35 @@ The migration process consists of two steps now:
    $ npm run migrate -- --rebuild
    ```
 
+### Skipping a dataSource for migration
+
+In some cases you do not want specific datasource(s) to be included in the
+migration process. For example, depending on the `process.env.NODE_ENV` you
+might be restricted to ALL or specific datasources from altering the table
+schemas.
+
+For such scenarios, you can include the property `allowMigration` in the
+datasource configuration. If you do not include it or if you set it to `true`
+then your datasource will be migrated, this is to provide a non-breaking change
+with this new property.
+
+The Datasource will only be skipped from the migration process if you set it to
+false.
+
+The example below shows how to do so in our Todo example application.
+
+{% include code-caption.html content="src/datasources/db.datasource.ts" %}
+
+```ts
+const config = {
+  name: 'db',
+  connector: 'memory',
+  localStorage: '',
+  file: './data/db.json',
+  allowMigration: false,
```

Comment:
Maybe `disableMigration` is better with default value to `false`.

Reply:
> Maybe `disableMigration` is better with default value to `false`.

Agree. @raymondfeng  PTAL.